app/src/main/java/com/heapixLearn/discovery/domain/contact/DumbServerContact.java

```diff
@@ -0,0 +1,35 @@
+package com.heapixLearn.discovery.domain.contact;
+
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.ServerStoreContact;
+
+public class DumbServerContact implements ServerStoreContact {
```

Comment:
@dianaprimshits unused class, should be removed

Reply:
@YodaMike Fixed